### ISSUE TITLE
Add screenreader text to the assessment's feedback strings

### ISSUE THREAD

**Issue by igorschoester** [Yoast/YoastSEO.js#2146]
## What did you expect to happen?

For accessibility, in every assessment's feedback string that opens a new tab/window. There should be a screenreader text before the end of the anchor tag that notifies people the link opens a new tab. We do this in our other repositories, but not here.

## What happened instead?

No screen reader text is present currently.

## Technical info

* Platform: WordPress | Drupal | Standalone
* Platform version:
* Yoast SEO version:

Which browser is affected (or browsers): 
- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Other 

If relevant, which editor is affected (or editors): 
- [ ] Classic Editor
- [ ] Gutenberg
- [ ] Classic Editor plugin

Beta
- [ ] Recalibration beta activated

**Comment by igorschoester** [comment]
Closing already, duplicate of https://github.com/Yoast/YoastSEO.js/issues/2131
